python.d/httpcheck.chart.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# Description: http check netdata python.d module
+# Original Author: ccremer (github.com/ccremer)
+
+import urllib3
+import time
+import re
+
+from bases.FrameworkServices.UrlService import UrlService
+
+try:
+    urllib3.disable_warnings()
+except AttributeError:
+    pass
```

Comment:
I see same code In [UrlSerice](https://github.com/firehol/netdata/blob/master/python.d/python_modules/bases/FrameworkServices/UrlService.py#L10-L13)
Are you sure that we need this here?